scribejava-ahc/src/main/resources/META-INF/services/com.github.scribejava.core.async.HttpProvider

```diff
@@ -0,0 +1 @@
+com.github.scribejava.core.async.ahc.AhcProvider
```

Comment:
"No newline at end of file."
It's common to add newline at all files, let it be so, thx